### PR TITLE
New subscription plan v2

### PR DESCRIPTION
**Pull request by shomix**

### CODE REVIEW

**Review by rekpero — CHANGES_REQUESTED** [review]
lgtm, please fix the nomenclature issue

File: src/components/Deployment/index.ts [hunk 1]
```diff
@@ -219,7 +208,9 @@ export async function subscriptionDeploymentFinished(
         memoryUsed,
         logs
       );
+    console.log("Update usage");
     await SubscriptionUsageService.updateUsage(deployment.id);
+    FindAndStartNextDeployment(deploymentId);
```
[the remark this answers is not in the document]
Comment (shomix) [reply src/components/Deployment/index.ts]: Done.

File: src/components/Deployment/index.ts [hunk 2]
```diff
@@ -374,7 +364,7 @@ export async function timedOut(req: Request, res: Response) {
       deployment.memoryUsed = 0;
       deployment.save();
     }
-    await SubscriptionUsageService.updateUsage(deployment.id);
+    await FindAndStartNextDeployment(deployment.id);
```
[the remark this answers is not in the document]
Comment (shomix) [reply src/components/Deployment/index.ts]: Done.

File: src/components/Deployment/index.ts [hunk 3]
```diff
@@ -916,7 +964,8 @@ function getLogsToCapture(protocol: string) {
       throw new Error(`Protocol ${protocol} not supported.`);
   }
 }
-async function startNextDeployment(deploymentId: string) {
+async function FindAndStartNextDeployment(deploymentId: string) {
```
[the remark this answers is not in the document]
Comment (shomix) [reply src/components/Deployment/index.ts]: Done.

**Pull request merged**